app/src/main/res/values/strings.xml

```diff
@@ -694,4 +694,54 @@
     <string name="sign_out_disconnect">Disconnect</string>
     <!-- Option to cancel signing out shown in confirmation dialog to sign out of account -->
     <string name="sign_out_cancel">Cancel</string>
+
+    <!-- Enhanced Tracking Protection -->
+    <!-- Link displayed in panel to access tracking protection settings -->
+    <string name="etp_settings">Protection Settings</string>
```

Comment:
Since this is the first occurrence of etp in the strings file (I think), changing the comment to say "enhanced tracking protection" would be nice to explain the acronym 😃